WordPress/src/main/java/org/wordpress/android/ui/accounts/helpers/FetchBlogListWPCom.java

```diff
@@ -44,6 +44,9 @@ protected void fetchBlogList(final Callback callback) {
                     site.put("isAdmin", jsonSite.get("user_can_manage"));
                     site.put("isVisible", jsonSite.get("visible"));
 
+                    // store capabilities as a text blob
```

Comment:
A minor nit, but a "blob" is typically an image or some other large object, so this comment might be confusing to future devs.